1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,36 @@
+let setNewColor = () => {
+  var randomColor = Math.floor(Math.random() * 16777215).toString(16);
+  return "#" + randomColor;
+};
+
+let content = document.getElementById("content");
+let pEl = document.createElement("p");
+let span = document.createElement("span");
+let button = document.createElement("button");
+let autoPlay = document.createElement("button");
+autoPlay.innerText = "auto-play:ON";
+autoPlay.id = "autoPlay";
+button.id = "addBtn";
+pEl.id = "item";
+span.id = "title";
+content.appendChild(pEl);
+content.appendChild(span);
+content.appendChild(button);
+content.appendChild(autoPlay);
+button.innerText = "New Quote";
+
+let newQuotes = (arr) => {
+  let newColor = setNewColor();
+  document.body.style.backgroundColor = newColor;
+  pEl.style.color = newColor;
+  autoPlay.style.color = newColor;
+  span.style.color = newColor;
+  button.style.color = newColor;
+  let quoteObj = pickFromArray(arr);
+  pEl.innerText = quoteObj.quote;
+  span.innerText = "- " + quoteObj.author;
+};
+
```

Comment:
I find this part of the code a little hard to read, maybe you could check out the style guide and see if you can update it to follow that?